### DISCUSSION TITLE
Would user interaction contaminate a uc mode user_data_dir?

### DISCUSSION THREAD

**Discussion by Dylgod** [seleniumbase/SeleniumBase#2784] [Q&A] [answered]
Lets say i ran this code:

```Python
try:
    driver = Driver(browser='chrome', uc=True, user_data_dir='path', headless=False, headed=True)
    driver.get('https://www.google.com')
    x = input("Press ENTER")
finally:
    driver.quit()
```

and found the elements i needed one by one for a test using the browser and never entering an input. Would i then be undetectable running a follow-up test with the same user-data-dir?

**Comment by mdmintz** [comment, marked as answer]
Unless you are in a UC Mode special breakpoint, you can be detected even when you are entering manual actions.
(See https://www.youtube.com/watch?v=2pTpBtaE7SQ&t=1196s)

As for whether or not you could be detected after performing certain actions and reusing the `user_data_dir`, you should experiment on the website you're using, and find out for yourself. It depends on multiple factors.
